app/models/story_parser.rb

```diff
@@ -989,8 +990,8 @@ def convert_revised_at(date_string)
           # probably seconds since the epoch
           date = Time.at($1.to_i)
         end
-        date ||= Date.parse(date_string)
-        return '' if date > Date.today
+        date ||= DateTime.parse(date_string)
+        return '' if date > DateTime.now
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.